emulators/dosbox-staging/Portfile

```diff
@@ -56,8 +56,8 @@ if {![variant_isset debug]} {
     configure.args-append -Dbuildtype=release
 }
 
-# Discourage Meson from downloading stuff during the configure process
-configure.args-append --wrap-mode=nofallback --wrap-mode=nodownload
```

Comment:
Bear in mind that Meson only appears to support one value for `--wrap-mode`, with the last one taking precedence.

So we don't lose anything with the new approach, and we achieve the same result as before.